frontend/src/pages/secret-manager/SecretDashboardPage/components/SecretListView/SecretItem.tsx

```
@@ -194,7 +208,23 @@
     };
 
     const handleFormSubmit = async (data: TFormSchema) => {
+      if (
+        importedBy &&
+        importedBy.some(({ folders }) =>
+          folders?.some(({ secrets }) =>
+            secrets?.some(({ referencedSecretId }) => referencedSecretId === secret.id)
+          )
+        )
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)